auth-lib/src/main/AndroidManifest.xml

```diff
@@ -44,7 +44,6 @@
         </activity>
 
         <activity
-            android:launchMode="singleInstance"
```

Comment:
I removed FLAG_ACTIVITY_SINGLE_TOP from createLoginActivityIntent because it prevents the system from re-creating the LoginActivity and calls onNewIntent instead, which we use to handle web redirects (and return the token/code).

This happens when you call openLoginActivity when LoginActivity is visible